packages/plugins/videos-ui/src/lib/shared/ui/video-actions/buttons/action-button/action-button.component.html

```
@@ -1,4 +1,13 @@
-<button nbButton class="action" [status]="button.status" [disabled]="button.disabled" (click)="triggerAction()">
+<button
+	nbButton
```

HTMLHint (1.5.0)

[error] The attribute name of [ nbButton ] must be in lowercase.

(attr-lowercase)